addon/components/flexberry-maptoolbar.js

```diff
@@ -13,11 +13,20 @@ export default Ember.Component.extend({
 
   activeToolName: null,
 
+  _isMeasureTool(toolname) {
+    return toolname && toolname.substr(-11) ===  'measuretool';
+  },
+
   actions: {
 
     activateTool(toolname) {
-      if(toolname === this.get('activeToolName')) {
-        return;
+      let activeToolName = this.get('activeToolName');
+      if (toolname === '+' && this._isMeasureTool(activeToolName)) {
```

Comment:
If i want to draw same (current) measure tool after cliking +-button it send toolname +